tests/test_basic_operation.py

```diff
@@ -102,25 +103,30 @@ def test_014_add_new_secondary():
     assert node3.wait_until_state(target_state="secondary")
     assert node2.wait_until_state(target_state="primary")
 
-
 def test_015_multiple_manual_failover_verify_replication_slot_removed():
-   monitor.failover()
-   assert node3.wait_until_state(target_state="primary")
-   assert node2.wait_until_state(target_state="secondary")
-   node2_replication_slots = node2.run_sql_query("select count(*) from pg_replication_slots")
-   assert node2_replication_slots == [(0,)]
-   node3_replication_slots = node3.run_sql_query("select count(*) from pg_replication_slots")
-   assert node3_replication_slots == [(1,)]
+    monitor.failover()
+    assert node3.wait_until_state(target_state="primary")
+    assert node2.wait_until_state(target_state="secondary")
+    node2_replication_slots = node2.run_sql_query("select count(*) from pg_replication_slots")
+    assert node2_replication_slots == [(0,)]
+    node3_replication_slots = node3.run_sql_query("select count(*) from pg_replication_slots")
+    assert node3_replication_slots == [(1,)]
    
-   monitor.failover()
-   assert node2.wait_until_state(target_state="primary")
-   assert node3.wait_until_state(target_state="secondary")
-   node2_replication_slots = node2.run_sql_query("select count(*) from pg_replication_slots");
-   assert node2_replication_slots == [(1,)]
-   node3_replication_slots = node3.run_sql_query("select count(*) from pg_replication_slots");
-   assert node3_replication_slots == [(0,)]
-                
+    monitor.failover()
+    assert node2.wait_until_state(target_state="primary")
+    assert node3.wait_until_state(target_state="secondary")
+    node2_replication_slots = node2.run_sql_query("select count(*) from pg_replication_slots");
+    assert node2_replication_slots == [(1,)]
+    node3_replication_slots = node3.run_sql_query("select count(*) from pg_replication_slots");
+    assert node3_replication_slots == [(0,)]
+
 def test_016_drop_primary():
-   node2.drop()
-   assert not node2.pg_is_running()
-   assert node3.wait_until_state(target_state="single")
+    node2.drop()
+    assert not node2.pg_is_running()
+    assert node3.wait_until_state(target_state="single")
+
+def test_017_stop_monitor():
```

Comment:
Mmm. There's a confusion around what process implements the monitor in our code and maybe even docs, and then users are confused to. I propose we name this test `test_017_stop_postgres_monitor` to help us get to a better consistent terminology here.

Reply:
ok